src/pages/PrivateNotes/PrivateNotesListPage.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {useMemo} from 'react';
+import {ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import withReportAndPrivateNotesOrNotFound from '@pages/home/report/withReportAndPrivateNotesOrNotFound';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {PersonalDetails, Report, Session} from '@src/types/onyx';
+
+type PrivateNotesListPageOnyxProps = {
+    /* Onyx Props */
```

Comment:
NAB: I didn't find this comment helpful could we remove it.

Reply:
I am seeing this for other files too, so I kept it.

Reply:
Looks like we don't need it for ts files since we're using `OnyxEntry` all other comments exist in non-ts files.